ya-erigon-runtime/src/main.rs

```diff
@@ -0,0 +1,208 @@
+use futures::channel::oneshot;
+use futures::FutureExt;
+use serde::{Deserialize, Serialize};
+use std::path::PathBuf;
+use std::process::Stdio;
+use std::sync::atomic::AtomicU64;
+use std::sync::atomic::Ordering::Relaxed;
+use tokio::process::{Child, Command};
+use ya_runtime_sdk::*;
+
+const ERIGON_BIN: &str = "tg";
+const RPCDAEMON_BIN: &str = "rpcdaemon";
+
+//TODO: Make parameter list configurable for further extendability (Erigon & Rpcdaemon)
+const RPCDAEMON_PARAMS: &[&str; 12] = &[
+    "--private.api.addr",
+    "localhost:9090",
+    "--http.addr",
+    "0.0.0.0",
+    "--http.port",
+    "8545",
+    "--http.vhosts",
+    "*",
+    "--http.corsdomain",
+    "*",
+    "--http.api",
+    "eth,debug,net,trace,web3,tg",
+];
+
+#[derive(Default, Deserialize, Serialize)]
+pub struct ErigonConf {
+    public_addr: Option<String>,
+    data_dir: Option<String>,
+}
+
+#[derive(Default, RuntimeDef)]
+#[conf(ErigonConf)]
+pub struct ErigonRuntime {
+    seq: AtomicU64,
+    erigon_pid: Option<Child>,
+    rpcdaemon_pid: Option<Child>,
+    host: Option<String>,
```

Comment:
Why is this named `host` not `puiblic_addr` like in `ErigonConf`? Do we even need this field? Config object is available in context so why copy the value into `ErigonRuntime`?

Reply:
I agree, its confusing. `Host` was meant to contain `public_addr + secret` ... (but it's not needed yet)
I'll remove it! :+1: